packages/jira-adapter/src/reference_mapping.ts

```diff
@@ -51,22 +51,59 @@ export const resolutionAndPriorityToTypeName: referenceUtils.ContextValueMapperF
   return undefined
 }
 
-export type ReferenceContextStrategyName = 'parentSelectedFieldType' | 'parentFieldType' | 'workflowStatusPropertiesIdContext' | 'workflowStatusPropertiesNameContext'
+export type ReferenceContextStrategyName = 'parentSelectedFieldType' | 'parentFieldType' | 'workflowStatusPropertiesContext'
 | 'parentFieldId'
 
 export const contextStrategyLookup: Record<
   ReferenceContextStrategyName, referenceUtils.ContextFunc
 > = {
   parentSelectedFieldType: neighborContextFunc({ contextFieldName: 'selectedFieldType', levelsUp: 1, contextValueMapper: toTypeName }),
   parentFieldType: neighborContextFunc({ contextFieldName: 'fieldType', levelsUp: 1, contextValueMapper: toTypeName }),
-  workflowStatusPropertiesIdContext: neighborContextFunc({ contextFieldName: 'key', contextValueMapper: getRefIdType }),
-  workflowStatusPropertiesNameContext: neighborContextFunc({ contextFieldName: 'key', contextValueMapper: getRefNameType }),
+  workflowStatusPropertiesContext: neighborContextFunc({ contextFieldName: 'key', contextValueMapper: getRefType }),
   parentFieldId: neighborContextFunc({ contextFieldName: 'fieldId', contextValueMapper: resolutionAndPriorityToTypeName }),
 }
 
-export const referencesRules: referenceUtils.FieldReferenceDefinition<
+const groupNameSerialize: GetLookupNameFunc = ({ ref }) =>
+  (ref.elemID.typeName === GROUP_TYPE_NAME ? ref.value.value.originalName : ref.value.value.id)
+
+type ReferenceSerializationStrategyName = 'groupStrategyById' | 'groupStrategyByOriginalName' | referenceUtils.ReferenceSerializationStrategyName
```

Comment:
nit: for consistency with the other cases, maybe add
```suggestion
type JiraReferenceSerializationStrategyName = 'groupStrategyById' | 'groupStrategyByOriginalName'
```
and combine with `referenceUtils.ReferenceSerializationStrategyName` in `JiraReferenceSerializationStrategyLookup`
?